cmd/cluster-init/generate_job.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	prowapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
+	prowconfig "k8s.io/test-infra/prow/config"
+
+	"github.com/openshift/ci-tools/pkg/api"
+)
+
+const (
+	LatestImage   = "registry.ci.openshift.org/ci/applyconfig:" + api.LatestReleaseName
+	OpenshiftRole = "ci.openshift.io/role"
```

Comment:
```suggestion
	LabelRole = "ci.openshift.io/role"
```

Everything is OpenShift :)

Reply:
![image](https://user-images.githubusercontent.com/1794300/132048428-a538c9c8-a2c6-4e55-9735-059352d47a38.png)